sabnzbd/config.py

```diff
@@ -786,7 +786,6 @@ def _read_config(path, try_backup=False):
 def save_config(force=False):
     """ Update Setup file with current option values """
     global CFG, database, modified
-    assert isinstance(CFG, configobj.ConfigObj)
```

Comment:
https://docs.python.org/2/reference/simple_stmts.html#the-assert-statement

assert is similar to throwing an exception if a given condition isn't true... so it's good to have and should be minimal hit for anything like checking if CFG is an instance before we save. An important difference is that assert statements get ignored if you compile your code with the optimization option. Since the `__debug__` variable gets set to false...  anyways the fact that we arent acting upon any of the asserts makes them pretty pointless however..  as shypike said, he uses them with WING ide.. thus you can leave them in but just add the if 0 statement before (which means it'd never run).. even if `__debug__` is set to true.

https://wingware.com/doc/edit/helping-wing-analyze-code
```
if 0:
    assert <whatever>
```